front/lib/resources/labs_transcripts_resource.ts

```diff
@@ -361,20 +378,24 @@ export class LabsTranscriptsConfigurationResource extends BaseResource<LabsTrans
     return history.get();
   }
 
-  async hasAnyHistory(): Promise<boolean> {
+  async hasAnyHistory(auth: Authenticator): Promise<boolean> {
+    const workspace = auth.getNonNullableWorkspace();
     const count = await LabsTranscriptsHistoryModel.count({
       where: {
         configurationId: this.id,
+        workspaceId: workspace.id,
```

Comment:
You should use this:
```suggestion
        workspaceId: this.workspaceId,
```